pkg/didcomm/crypto/legacy/authcrypt/decrypt.go

```diff
@@ -7,16 +7,165 @@ SPDX-License-Identifier: Apache-2.0
 package authcrypt
 
 import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
 	"errors"
+	"fmt"
 
+	"github.com/btcsuite/btcutil/base58"
+	chacha "golang.org/x/crypto/chacha20poly1305"
+	"golang.org/x/crypto/ed25519"
```

Comment:
now that the linter supports Go 1.13, let's try to use the internal crypto package instead..

Reply:
I've switched `crypto/curve25519` to import from the standard library instead of `golang.org/x/crypto`. The others I use have not been moved over yet.